<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_data_dialog</class>
 <widget class="QDialog" name="new_data_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="new_data_save">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(185, 187, 223);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
  </widget>
  <widget class="QPushButton" name="new_data_cancel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(185, 187, 223);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowClose"/>
   </property>
  </widget>
  <widget class="QTextEdit" name="new_data_code_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="tabStopDistance">
    <double>40.000000000000000</double>
   </property>
  </widget>
  <widget class="QTextEdit" name="new_data_comment_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>321</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabStopDistance">
    <double>40.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_data_zh_index">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_data_en_index">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(31, 171, 137);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;zh_index&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(31, 171, 137);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;en_index&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="language_switch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(31, 171, 137);</string>
   </property>
   <item>
    <property name="text">
     <string>C</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C++</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rust</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Go</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Python</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lisp</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Assembly</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Java</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JavaScript</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HTML</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CSS</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(31, 171, 137);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;language&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_data_delete">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(185, 187, 223);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(31, 171, 137);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;number_index&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_data_number_index">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>new_data_number_index</tabstop>
  <tabstop>new_data_zh_index</tabstop>
  <tabstop>new_data_en_index</tabstop>
  <tabstop>new_data_code_edit</tabstop>
  <tabstop>new_data_comment_edit</tabstop>
  <tabstop>language_switch</tabstop>
  <tabstop>new_data_save</tabstop>
  <tabstop>new_data_delete</tabstop>
  <tabstop>new_data_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
